_sass/_page.scss

```diff
@@ -55,8 +55,15 @@
     border-bottom: 1px solid $border-color;
   }
 
+  h1, h2, h3, h4, h5, h6 {
+    font-family: "Roboto", sans-serif;
+  }
+
   p, li, dl {
-    font-size: 1em;
+    font-size: 18px;
+    font-weight: 300;
+    line-height: 28px;
+    font-family: "Roboto-Light", sans-serif
```

Comment:
These style changes set the font styles to match the other major IOOS themed sites.

![screen shot 2016-10-05 at 4 23 19 pm](https://cloud.githubusercontent.com/assets/1155163/19130150/0d62f618-8b18-11e6-917c-f8dc1c6bca17.png)